### PR TITLE
refactor(client): guard clauses and DropFingerprint

### PR DESCRIPTION
**Pull request by mafewtm**
## Description

- Addition of main.lua so we can begin properly seperating things
- Creates the DropFingerprint function
- Addition of guard clauses in the use item events

## Checklist

- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [ ] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

File: config/client.lua [hunk 1]
```diff
@@ -1,6 +1,8 @@
 return {
     useTarget = false,
 
+    fingerprintChance = 50, -- Chance of dropping a fingerprint if not wearing gloves
```
Comment (Manason) [inline config/client.lua:4]: Optional; Would this be better as 0.5 then 50? As in, is it better to express percentages as decimals?

File: client/main.lua [hunk 2]
```diff
@@ -0,0 +1,10 @@
+local config = require 'config.client'
+
+function DropFingerprint()
+    if IsWearingGloves() then return end
+
+    local coords = GetEntityCoords(cache.ped)
```
Comment (Manason) [inline client/main.lua]: move the coords var inside the if
https://manason.github.io/effective-fivem-lua/variables/#location

**Review by Manason — CHANGES_REQUESTED** [review]
Seems like the difference banks have very similar logic. Are there function(s) that could be extracted to share duplicate logic?

File: config/client.lua (hunk 1 above)
Comment (mafewtm) [reply config/client.lua:4]: I have no idea

**Review by Manason — APPROVED** [review]

File: config/client.lua (hunk 1 above)
Comment (solareon) [reply config/client.lua:4]: If it was 0.5 then the math.random needs to be changed to just () vice specifying the int values. From a _config dev_ point of view the 50 is clearer to understand.

**Review by solareon — APPROVED** [review]

**Pull request merged**